lib/ember_cli/build_monitor.rb

```diff
@@ -36,7 +36,11 @@ def complete?
     end
 
     def build_error?
-      error_file.exist? && error_file.size?
+      error_file.exist? && error_file.size? && not(has_deprecation_warnings?)
```

Comment:
Also look at `EmberCli::BuildMonitor#reset!` - this method is used to empty the error file between builds. Internally it uses `build_error?`, but even when a deprecation warning is seen it will still need to be deleted. The method needs to be refactored to delete the file regardless of whether the contents are deprecation warnings or build errors.